ncurses/PKGBUILD

```diff
@@ -92,9 +92,12 @@ package_ncurses-devel() {
   rm -rf ${pkgdir}/usr/lib/terminfo
 
   cp -rf ${pkgdir}/usr/include/ncursesw ${pkgdir}/usr/include/ncurses
+  cp -f ${pkgdir}/usr/include/ncursesw/*.h ${pkgdir}/usr/include
```

Comment:
This is also done in cygwin (see [here](https://github.com/cygwinports/mingw64-x86_64-ncurses/blob/master/mingw64-x86_64-ncurses.cygport#L37). The headers are installed in /usr/include, and links are created in ncurses and ncursesw). I don't know how many project include these headers without the `ncurses/` prefix, but at least sipp does.